src/glaciers/climate/climate2D_utils.jl

```diff
@@ -105,6 +193,18 @@ function get_raw_climate_data(rgi_path::String)
 end
 
 # TODO: make snow/rain thresholds customizable
+"""
+    apply_t_cumul_grad!(climate_2D_step::Climate2Dstep, S::Matrix{F}) where {F <: AbstractFloat}
+
+Apply temperature and precipitation degree day (PDD) gradients to the climate data in `climate_2D_step` based on the elevation matrix `S`.
```

Comment:
The use of PDD and "precipitation degree day" is confusing here. Maybe something like that?
```suggestion
Apply temperature and precipitation gradients based on the positive degree day (PDD) and on the elevation matrix `S` to the climate data in `climate_2D_step`.
```

Reply:
You're right, that was a hallucination.